cuppa/src/main/java/org/forgerock/cuppa/Runner.java

```diff
@@ -123,9 +125,8 @@ private TestBlock defineTestsWithConfiguration(Iterable<Class<?>> testClasses, T
     }
 
     private TestBlock mergeRootTestBlocks(TestBlock testBlock1, TestBlock testBlock2) {
-        return new TestBlock(ROOT, NORMAL, Cuppa.class, "", Stream.concat(testBlock1.testBlocks.stream(),
-                testBlock2.testBlocks.stream()).collect(Collectors.toList()), Collections.emptyList(),
-                Collections.emptyList(), Options.EMPTY);
+        return EMPTY_TEST_BLOCK.toBuilder().setTestBlocks(Stream.concat(testBlock1.testBlocks.stream(),
```

Comment:
Interesting idea. I think I'd like to think about it more carefully. Can we take this discussion offline and consider it for later inclusion?